medusa/search/core.py

```diff
@@ -906,6 +906,15 @@ def combine_results(multi_results, single_results):
     """Combine single and multi-episode results, filtering out overlapping results."""
     log.debug(u'Combining single and multi-episode results')
     result_candidates = []
+    return_single_results = []
+    return_multi_results = []
+
+    # Only return one result per episode for the single results and multi-result result_candidates
+    if single_results:
```

Comment:
@medariox this is now added to go back from a list like: `[61, 52, 61, 11, 11, 61, 61]` to `[61, 52, 11]` (the numbers are now episode objects of course, which can be compared to one another.